cmd/config/loader.go

```diff
@@ -34,51 +35,99 @@ func GetOutputConfiguration() OutputConfiguration {
 	return configuration.Output
 }
 
-func mustLoadConfig() config {
-	configData, err := loadConfig()
+func LoadDefaultConfig() error {
+	var configData config
+	err := loadConfigFromYaml(yamlDefaultData, &configData)
+	if err != nil {
+		return fmt.Errorf("could not load default config: %s", err)
+	}
+
+	err = configData.Validate()
 	if err != nil {
-		panic("Could not load config: " + err.Error())
+		return fmt.Errorf("invalid default config: %s", err)
 	}
-	return configData
+
+	configuration = configData
+	return nil
 }
 
-func loadConfig() (config, error) {
+func LoadConfig(navigator Navigator) error {
+	// 1. default config from repo
```

Comment:
took me a second to understand it :D 


```suggestion
	// 1. default config from ".tfcoach.default.yml"
```